addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -13,6 +13,9 @@
     <xpath expr="//div[@id='wrapwrap']" position="attributes">
         <attribute name="t-att-data-forum_id">forum and forum.id</attribute>
     </xpath>
+    <xpath expr="//body" position="inside">
+        <script t-if="seo_microdata" type="application/ld+json"><t t-out="seo_microdata"/></script>
+    </xpath>
```

Comment:
I see in the [Google example](https://developers.google.com/search/docs/appearance/structured-data/discussion-forum#json-ld%20example) that they put this script tag in the `<head>`. Is there a reason you add it in the body ?

Reply:
It's the opposite: the `<head>` is not t-cached (except for website.pages)

https://github.com/odoo/odoo/blob/de8f82b883829c250292c2ad0d8710e0ddbb7911/addons/website/views/website_templates.xml#L72-L87

It's why I was worried about adding it in `<head/>` as it would perform all the SQL query all the time. While if it's in the blog post page, it should be cached.

As discussed in DM:
- We have perf test for /blog but not for /blog/post sadly it seems?
- So we have to triple check that adding the micro data does not skyrocket the perfs 
- I would take the opportunity to add some sql perf on /blog/post is there is none (confirm what I quickly checked?)

Reply:
I suppose you mean forum and not blog @rdeodoo 

But indeed, we need to know how much sql request this PR adds to render a forum page @SergeBayet 
Add a perf test commit before your commit and try to reduce the nb of sql request of your IMP.

Reply:
@Guillaume-gdi @rdeodoo Could you tell me if it seems good to you? I've added a performance test.